tests/classes/PluginManager.js

```diff
@@ -765,8 +766,11 @@ describe('PluginManager', () => {
   describe('Plugin/CLI integration', () => {
```

Comment:
This is actually running a 'test' inside a 'describe' block. This should probably be wrapped in an `it(...)`, no?

(I realize this isn't related to what you're doing, but I was going to fix it in a separate PR but it would step on your changes)